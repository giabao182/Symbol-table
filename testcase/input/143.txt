INSERT a number
INSERT b string
ASSIGN b 'hello'
ASSIGN a b