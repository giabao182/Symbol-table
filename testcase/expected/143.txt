success
success
success
TypeMismatch: ASSIGN a b